app/serializers/data_explorer/query_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class DataExplorer::QuerySerializer < ActiveModel::Serializer
+  attributes :id, :sql, :name, :description, :param_info, :created_at, :username, :group_ids, :last_run_at, :hidden, :user_id
+
+  def param_info
+    object.params.map(&:to_hash) rescue nil
+  end
+
+  def username
+    object.user.username rescue nil
+  end
+
+  def group_ids
+    object.groups.map(&:id)
```

Comment:
Hmm it doesn't seem like we're eager loading `groups` anywhere so this might cause an N+1?